cardano-node/test/Test/Cardano/Node/Chairman.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Test.Cardano.Node.Chairman
+  ( tests
+  ) where
+
+import           Cardano.Prelude
+import           Hedgehog (Property, discover)
+
+import qualified Hedgehog as H
+import qualified System.Process as IO
+import qualified Test.Common.Base as H
+import qualified Test.Common.Process as H
+
+prop_spawnOneNode :: Property
+prop_spawnOneNode = H.propertyOnce $ do
+  (_mIn, _mOut, _mErr, hProcess) <- H.createProcess $
```

Comment:
Nice! Not sure what direction you are going in here but I would create a function that accepts a record type parameterizing the database path, socket path etc. Not necessarily in this PR though.

Reply:
Sounds good.  I'll look into that.